cmd/private-org-sync/main.go

```diff
@@ -351,9 +342,13 @@ func (g gitSyncer) mirror(repoDir string, src, dst location) error {
 		}
 
 		if depth == unshallow {
-			message := "failed to push to destination, no retry possible (already fetched full history)"
-			logger.Error(message)
-			return nil, fmt.Errorf(message)
+			logger.Info("Trying to fetch source and destination full history and perform a merge")
+			destRemoteURI := fmt.Sprintf("https://github.com/%s/%s", dst.org, dst.repo)
```

Comment:
Please reuse `destUrl`: there's no point in building the URL twice, plus `destUrl` already contains the necessary token to access the (usually private) repo.

Reply:
addressed